CHANGELOG.md

```diff
@@ -1,6 +1,110 @@
 ## 2.15.0 (TBC)
 
-Changes in progress for v2.15.0 are available at [.changes/v2.15.0](https://github.com/vmware/go-vcloud-director/tree/main/.changes/v2.15.0) until the release.
+## FEATURES
+* Added support for Shareable disks, i.e., independent disks that can be attached to multiple VMs which is available from
+  API v35.0 onwards. Also added uuid to the Disk structure which is a new member that is returned from v36.0 onwards. This
+  member holds a UUID that can be used to correlate the disk that is attached to a particular VM from the VCD side and the
+  VM host side. ([#383](https://github.com/vmware/go-vcloud-director/pull/383))
+* Added support for uploading OVF using URL `catalog.UploadOvfByLink` ([#422](https://github.com/vmware/go-vcloud-director/pull/422)], [[#426](https://github.com/vmware/go-vcloud-director/pull/426))
+* Added support for updating vApp template `vAppTemplate.UpdateAsync` and `vAppTemplate.Update` ([#422](https://github.com/vmware/go-vcloud-director/pull/422))
+* Add methods `catalog.PublishToExternalOrganizations` and `adminCatalog.PublishToExternalOrganizations` ([#424](https://github.com/vmware/go-vcloud-director/pull/424))
```

Comment:
`Add` and `Added`are used interchangeably.  Maybe we should use one or another, not both to keep consistency?

Reply:
Made past tense everywhere